### DISCUSSION TITLE
logged password in  plain text

### DISCUSSION THREAD

**Discussion by smile0ai** [rabbitmq/rabbitmq-server#3211] [Other]
Hi guys maybe this issue is not actual but i seen in our  logs  rabit mq login and password in plain text how can i shadow it?
i found https://github.com/rabbitmq/rabbitmq-server/issues/1837  it the actual? use RBAC for logs? 
or maybe i can shadow password in logs?
with best regards and be healthy.
..amqp_params_network,<<"user1">>,
                                <<"pass1">>,<<"/">>,
                                "host1",5672
..

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
You haven't provided any details, e.g. any actual stack traces in the logs or RabbitMQ version, for us to work with.

Modern versions of the Erlang client (and thus the plugins that use it, like Shovel and Federation) [encrypt endpoint password in process state](https://github.com/rabbitmq/credentials-obfuscation/) to avoid logging it. This has been
the case for [close to two years](https://github.com/rabbitmq/rabbitmq-erlang-client/pull/124).

We are not aware of any other scenarios where a logged exception could log those values. If they are logged, they are
not actual passwords but rather their encrypted values (the key is a one-off, specific to the cluster).
